dot/core/service_test.go

```diff
@@ -24,16 +24,29 @@ import (
 	"github.com/ChainSafe/gossamer/lib/transaction"
 	"github.com/ChainSafe/gossamer/lib/trie"
 	"github.com/ChainSafe/gossamer/pkg/scale"
-	cscale "github.com/centrifuge/go-substrate-rpc-client/v3/scale"
-	"github.com/centrifuge/go-substrate-rpc-client/v3/signature"
-	ctypes "github.com/centrifuge/go-substrate-rpc-client/v3/types"
+	cscale "github.com/centrifuge/go-substrate-rpc-client/v4/scale"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/signature"
+	ctypes "github.com/centrifuge/go-substrate-rpc-client/v4/types"
 
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 var errTestDummyError = errors.New("test dummy error")
+var testAPIItem = runtime.APIItem{
+	Name: [8]byte{0xd2, 0xbc, 0x98, 0x97, 0xee, 0xd0, 0x8f, 0x15},
+	Ver:  3,
+}
+var runtimeVersionData = runtime.NewVersionData(
+	[]byte("polkadot"),
+	[]byte("parity-polkadot"),
+	authoringVersion,
+	specVersion,
+	implVersion,
+	[]runtime.APIItem{testAPIItem},
+	transactionVersion,
+)
```

Comment:
having them inlined / locally defined as it was, was better imo.
Better to copy paste some code rather than having dependency and funny mutation surprises